RecoEgamma/EgammaElectronProducers/plugins/GEDGsfElectronFinalizer.cc

```diff
@@ -103,6 +105,12 @@ void GEDGsfElectronFinalizer::produce( edm::Event & event, const edm::EventSetup
      isoVariables.sumPhotonEt = (*(isolationValueMaps)[1])[myElectronRef];
      isoVariables.sumNeutralHadronEt = (*(isolationValueMaps)[2])[myElectronRef];
      isoVariables.sumPUPt = (*(isolationValueMaps)[3])[myElectronRef];
+     if(isolationValueMaps.size()<=4){
```

Comment:
How is this different from checking ```nDeps_``` ?

Reply:
My apologies, this was some debugging code left in accidentally, its completely unnecessary and should be removed. I will do this shortly.